src/main/scala/ru/org/codingteam/horta/database/PersistentStore.scala

```diff
@@ -27,6 +27,7 @@ class PersistentStore(repositories: Map[String, RepositoryFactory]) extends Acto
   var initializedSchemas = Set[String]()
 
   override def preStart() {
+    GlobalSettings.loggingSQLAndTime = GlobalSettings.loggingSQLAndTime.copy(singleLineMode = true)
```

Comment:
I think that it is not nice to reset `GlobalSettings.loggingSQLAndTime` any time we are starting a storage actor. It would be better to move this line to the `Application` class.